functions/create_draft/interactivity_handler.ts

```diff
@@ -73,6 +74,40 @@ export const openDraftEditView: BlockActionHandler<
       });
     }
   }
+  // If the user selects to discard the draft message
+  if (action.selected_option.value == "discard_message_overflow") {
+    const id = action.block_id;
+    const thread_ts = body.message?.ts || "";
+
+    // Delete the draft message from the Draft Channel
+    const updateResp = await client.chat.delete({
+      channel: inputs.channel,
+      ts: thread_ts,
+    });
+
+    if (!updateResp.ok) {
+      const updateDraftPreviewErrorMsg =
+        `Error deleting the draft message: ${thread_ts} in channel ${inputs.channel}. Contact the app maintainers with the following - (Error detail: ${updateResp.error})`;
+      console.log(updateDraftPreviewErrorMsg);
```

Comment:
Nit: `console.error` might be preferable here for those redirected outputs. Not a blocker though!

Reply:
💯 Will update these ones that are under the '!' Responses to `console.error` instead 👌